wolfcrypt/src/error.c

```diff
@@ -601,6 +601,9 @@ const char* wc_GetErrorString(int error)
     case AES_EAX_AUTH_E:
         return "AES-EAX Authentication check fail";
 
+    case KEY_EXHAUSETED_E:
```

Comment:
I like this idea for an error code. Typo though:
`KEY_EXHAUSETED_E` should be `KEY_EXHAUSTED_E`

Reply:
Fixed